src/esm.ts

```diff
@@ -65,6 +65,58 @@ export function genDynamicImport (specifier: string, opts: DynamicImportOptions
   return `${wrapperStr}import(${genString(specifier, opts)}${commentStr})${ineropStr}`
 }
 
-export function genImportName (name: string) {
+export function genSafeVariableName (name: string) {
+  if (reservedNames.includes(name)) { return `_${name}` }
   return name.replace(/^[^_a-zA-Z]|[^\w]/g, r => '_' + r.charCodeAt(0))
 }
+
+const reservedNames = [
```

Comment:
Perf: We might use a Set
Style: Can we use multi-line with multi values to make code shorter?

Reply:
Credit: I would add a comment for the reference :)